test/integration/fleetscope/fleetscope_test.go

```diff
@@ -23,10 +23,97 @@ import (
 	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
 	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
 	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/gruntwork-io/terratest/modules/shell"
 	"github.com/stretchr/testify/assert"
 	"github.com/terraform-google-modules/enterprise-application/test/integration/testutils"
 )
 
+func retrieveNamespace(t *testing.T) (string, error) {
+	cmd := "get ns config-management-system"
+	args := strings.Fields(cmd)
+	kubectlCmd := shell.Command{
+		Command: "kubectl",
+		Args:    args,
+	}
+	return shell.RunCommandAndGetStdOutE(t, kubectlCmd)
```

Comment:
For `kubectl` I recommend using `"github.com/gruntwork-io/terratest/modules/k8s"`:

```
k8sOpts := k8s.KubectlOptions{}
configNameSpace, err := k8s.RunKubectlAndGetOutputE(t, &k8sOpts, "get", "ns", "config-management-system", "-o", "json")
assert.NoError(err)
```

If you are testing (get credentials) in parallel, you can also specify the exact cluster content
```
k8sOpts := k8s.NewKubectlOptions(fmt.Sprintf("gke_%s_%s_%s", projectId, location, clusterName), "", "")
```